JSIL.Libraries/Sources/JSIL.Core.js

```diff
@@ -5635,6 +5635,10 @@ JSIL.MakeType = function (typeArgs, initializer) {
       };
     } else {
       typeObject._IsAssignableFrom = function (typeOfValue) {
+        if (!typeOfValue.__TypeInitialized__ && true) {
```

Comment:
Probably too many work :) Really it is nearly same as ||false, but will
change to common form tomorrow.
On Oct 21, 2015 1:15 AM, "Katelyn Gadd" <notifications@github.com> wrote:

> In JSIL.Libraries/Sources/JSIL.Core.js
> <https://github.com/sq/JSIL/pull/879#discussion_r42594539>:
>
> > @@ -5635,6 +5635,10 @@ JSIL.MakeType = function (typeArgs, initializer) {
> >        };
> >      } else {
> >        typeObject._IsAssignableFrom = function (typeOfValue) {
> > +        if (!typeOfValue.__TypeInitialized__ && true) {
>
> Any reason why the true is here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sq/JSIL/pull/879/files#r42594539>.
>